<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProject</class>
 <widget class="QWidget" name="CreateProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateProject</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#CreateProject {
	background-color: rgb(50, 50, 50);
}
* {
	font-family: &quot;Arial&quot;;
	font-weight: bold;
}

QLabel {
	font-weight: bold;
	color: rgb(200, 200, 200);
}

QCheckBox {
	color: rgb(225, 225, 225);
	font: 500 8pt &quot;Arial&quot;;
}
QPushButton, QLineEdit, QSpinBox {
	border: 1px solid rgb(75, 75, 75);
	background-color: rgb(35, 35, 35);
	color: rgb(200, 200, 200);
}
QPushButton:hover, QLineEdit:focus, QSpinBox:focus {
	border-color: rgb(0, 128, 255);
	color: rgb(255, 255, 255);
}
QPushButton:hover:pressed {
	background-color: rgb(25, 25, 25);
}</string>
  </property>
  <widget class="QWidget" name="w_dlg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget {
	background-color: rgb(40, 40, 40);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="create_btn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spin_width">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>1920</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spin_height">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>1080</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="line_name">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-weight: bold;
color: white;
font-family: &quot;Arial&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>New project</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
